listenbrainz/webserver/static/js/src/components/Pill.tsx

```diff
@@ -12,9 +18,9 @@ type PillProps = {
 export default function Pill(props: React.PropsWithChildren<PillProps>) {
   const { active, children, type, style: propStyle, ...buttonProps } = props;
 
-  let background = "#EB743B";
+  let background = COLOR_LB_ORANGE;
```

Comment:
Can't help but feel that in this case we could do with moving all these styles to CSS instead, with appropriate states (`.active`, `.secondary`, etc.)

Reply:
I agree with this. Using `CSSProperties` is uncommon in LB codebase and styles should move to a CSS file.
I'm adding it to my list :)